pkg/destroy/aws/ec2helpers.go

```diff
@@ -786,7 +786,39 @@ func deleteEC2VPCPeeringConnection(ctx context.Context, client *ec2.EC2, id stri
 }
 
 func deleteEC2VPCEndpointService(ctx context.Context, client *ec2.EC2, id string, logger logrus.FieldLogger) error {
-	_, err := client.DeleteVpcEndpointServiceConfigurationsWithContext(ctx, &ec2.DeleteVpcEndpointServiceConfigurationsInput{
+	output, err := client.DescribeVpcEndpointConnectionsWithContext(ctx, &ec2.DescribeVpcEndpointConnectionsInput{
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("service-id"),
+				Values: aws.StringSlice([]string{id}),
+			},
+		},
+	})
+
+	if err != nil {
+		logger.Warn("Unable to get the list of VPC endpoint connections connected to service: ", err)
+		logger.Warn("Attempting to delete the VPC Endpoint Service")
+	} else {
+		endpointList := make([]*string, len(output.VpcEndpointConnections))
+		for _, endpoint := range output.VpcEndpointConnections {
+			if aws.StringValue(endpoint.VpcEndpointState) != "rejected" {
+				endpointList = append(endpointList, endpoint.VpcEndpointId)
+			}
+		}
+
+		_, err = client.RejectVpcEndpointConnectionsWithContext(ctx, &ec2.RejectVpcEndpointConnectionsInput{
+			ServiceId:      &id,
+			VpcEndpointIds: endpointList,
+		})
+
+		if err != nil {
+			logger.Warn("Unable to reject VPC endpoint connections for service: ", err)
+			logger.Warn("Attempting to delete the VPC Endpoint Service")
+		}
+		logger.WithField("resourceType", "VPC Endpoint Connection").Info("Rejected")
```

Comment:
```suggestion
		} else {
		    logger.WithField("resourceType", "VPC Endpoint Connection").Info("Rejected")
		}
```
otherwise you're going to log that you could not reject the connection and that it was rejected.